ts/parts-more/BoxPlotSeries.ts

```diff
@@ -464,8 +514,11 @@ seriesType<Highcharts.BoxPlotSeries>('boxplot', 'column', {
                         options.stemWidth,
                         options.lineWidth
                     );
-                    stemAttr.dashstyle =
-                        point.stemDashStyle || options.stemDashStyle;
+                    stemAttr.dashstyle = pick(
+                        point.stemDashStyle,
+                        options.stemDashStyle,
+                        options.dashStyle
+                    );
```

Comment:
Not a problem, just a hint: in general `pick()` is useful when dealing with numerical values, where `a || b` can not be used (as `a=0` would pick `b` value).